fc-exploitation/views/service-provider/creation.ejs

```diff
@@ -216,20 +216,21 @@
                 <select class="custom-select" name="id_token_signed_response_alg" required>
                   <option value="HS256" <%= messages.values && messages.values[0].id_token_signed_response_alg === "HS256" ? 'selected' : '' %>>HS256</option>
                   <option value="ES256" <%= messages.values && messages.values[0].id_token_signed_response_alg === "ES256" ? 'selected' : '' %>>ES256</option>
-                  <option value="RS256" <%= messages.values && messages.values[0].userinfo_signed_response_alg === "RS256" ? 'selected' : '' %>>RS256</option>
+                  <option value="RS256" <%= messages.values && messages.values[0].id_token_signed_response_alg === "RS256" ? 'selected' : '' %>>RS256</option>
                 </select>
```

Comment:
:praise Nice fix, thanks! We can maybe share with FranceConnect team.

Reply:
Has been shared with France Connect team on [Tchap](https://tchap.gouv.fr/#/room/!pXgBuqVpNnDdLrLvCa:agent.dinum.tchap.gouv.fr/$p6ijUg1gHS__w2jma4ydteGpHbMAXEiInZQxcawnjjo?via=agent.dinum.tchap.gouv.fr&via=agent.dev-durable.tchap.gouv.fr)